base/tekton.dev/tasks/build-layer.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: build-layer
+spec:
+  steps:
+    - image: quay.io/podman/stable:latest
+      name: build-layer
+      resources: {}
+      securityContext:
+        privileged: true
+      script: |
+        #!/usr/bin/env bash
+        set -euxo pipefail
+        
+        dnf install -y jq
+        
+        cd /srv/coreos
+
+        OSTREE_VERSION=$(jq -r '."ostree-version"' < builds/latest/$(uname -m)/meta.json)
+        OSTREE_TARFILE=$(jq -r '.images.ostree.path' < builds/latest/$(uname -m)/meta.json)
+
+        # the c9s.repo file is meant to be consumed by a COSA container
+        # where the RPM GPG keys are shipped in /usr/share/distribution-gpg-keys/centos
+        # SCOS however ships the keys in the /etc/pki/rpm-gpg
+        sed -i 's;/usr/share/distribution-gpg-keys/centos;/etc/pki/rpm-gpg;g' src/config/c9s.repo
+        sed -i 's/RPM-GPG-KEY-CentOS-Official/RPM-GPG-KEY-centosofficial/g' src/config/c9s.repo
+
+        cat <<EOF >> src/config/Dockerfile
+        FROM quay.io/okd/centos-stream-coreos-9:4.14-x86_64
+        COPY rpms/ /srv/coreos/rpms/
+        COPY src/config/c9s.repo /etc/yum.repos.d/c9s.repo
+        COPY src/config/packages-openshift.yaml /etc/rpm-ostree/origin.d/packages-openshift.yaml
+        RUN cat /etc/os-release \
+            && rpm-ostree --version \
+            && ostree --version \
+            && rpm-ostree ex rebuild \
+            && rpm-ostree cleanup -m \
+            && rm -rf /var/lib/unbound /srv/coreos/rpms /etc/rpm-ostree/origin.d /etc/yum.repos.d/c9s.repo \
+            && ostree container commit
+
+        LABEL io.openshift.release.operator=true \
+              io.openshift.build.version-display-names="machine-os=CentOS Stream CoreOS" \
+              io.openshift.build.versions="machine-os=${OSTREE_VERSION}"
+
+        EOF
+
+        ls /srv/coreos/builds/latest/$(uname -m)/${OSTREE_TARFILE} 
+        podman build \
+          -f src/config/Dockerfile \
+          -t ostree-okd \
+          .
+        # mkdir /srv/ostree-okd
+        podman save --format oci-dir ostree-okd:latest -o /srv/ostree-okd
+        # podman tag localhost/ostree-okd quay.io/okd/stream-coreos-okd
+        # podman push  quay.io/okd/stream-coreos-okd
+        # TODO Allow for pulling previously built base image, and for pushing the layered image
+        printf "%s" "OK" > $(results.output.path)
+  results:
+    - name: output
+  workspaces:
+    - mountPath: /srv
```

Comment:
For consistency with https://github.com/okd-project/okd-coreos-pipeline/commit/067ce9d787164487f6a9d5cab770a37b1dd1b99d we should use `/workspace`

Reply:
And, replace all the references of `/srv` by `/workspace` in the script